yarn.lock

```diff
@@ -1328,11 +1328,6 @@
   dependencies:
     "@types/node" "*"
 
-"@types/xmldom@^0.1.29":
-  version "0.1.29"
-  resolved "https://registry.yarnpkg.com/@types/xmldom/-/xmldom-0.1.29.tgz#c4428b0ca86d3b881475726fd94980b38a27c381"
-  integrity sha1-xEKLDKhtO4gUdXJv2UmAs4onw4E=
-
```

Comment:
I think this was previously introduced by error but is no longer depended on. It's unrelated to the change at hand, but it's small enough I don't mind bundling.